doc/classes/DisplayServer.xml

```diff
@@ -1753,17 +1753,17 @@
 			[b]Note:[/b] This flag is implemented on macOS.
 		</constant>
 		<constant name="VSYNC_DISABLED" value="0" enum="VSyncMode">
-			No vertical synchronization, which means the engine will display frames as fast as possible (tearing may be visible). Framerate is unlimited (nonwithstanding [member Engine.max_fps]). Not supported when using the Compatibility rendering method.
```

Comment:
@clayjohn just for personal knowledge, does OpenGL not have v-sync or is it not possible to turn off. I kind of need to make sure v-sync is off :P

Reply:
never mind, you removed the remark :)